### ISSUE TITLE
Navigation links: Part of the focus outline is hidden

### ISSUE THREAD

**Issue by vipul0425** [rtCamp/godam#535]
##  Issue

* On the plugin’s settings page, when a menu link is focused using the keyboard, the focus outline is hidden due to overlap with the active menu item.
* The focus ring on footer social icons appears misaligned when navigated via keyboard.

---

## ✅ Expected Behavior

* The focus ring should be clearly visible and properly aligned with the focused element to maintain accessibility and usability.

## Screenshots

<img width="1448" alt="Image" src="https://github.com/user-attachments/assets/dfb43ac7-9f5e-415c-811d-1ba69c330c3e" />

<img width="786" alt="Image" src="https://github.com/user-attachments/assets/f436a2c5-196c-4ce6-b0cb-7d7926462d48" />

<img width="582" alt="Image" src="https://github.com/user-attachments/assets/20d26b94-ce74-40b1-a621-08bdda3701ed" />

cc: @nayemDevs

**Comment by KMchaudhary** [comment]
Thanks @vipul0425 for fixing this navigation outline style issue.
I approved and merged the PR #539
